src/cosmic/src/kick.f

```diff
@@ -207,24 +213,33 @@ SUBROUTINE kick_pfahl(kw,m1,m1n,m2,ecc,sep,jorb,vk,sn,r2,
           xx = RAN3(idum1)
           xx = RAN3(idum1)
       else
+* If no pre-supplied kick magnitude, we draw a kick from a distribution
+* If the kickflag is 5 then use the log-normal distribution described
+* by Disberg & Mandel 2025
+          if(abskickflag.eq.5.and..not.ECSN_or_USSN)then
```

Comment:
It looks like `ECSN_or_USSN` is only used with Disberg+; just double checking: should we be applying a similar flag for the other kickflags?

Reply:
Got it -- and yeah, this makes sense!